webapp/components/create_post.jsx

```diff
@@ -107,13 +116,36 @@ export default class CreatePost extends React.Component {
         }
     }
 
+    handleHide() {
+        this.setState({
+            showConfirmModal: false
+        });
+    }
+
+    handleNotifyAllConfirmation() {
+        this.setState({
+            showConfirmModal: false
+        });
+        this.sendMessage(this.state.postMessage);
```

Comment:
Instead of duplicating the logic to send the message and reset the state of the textbox, can you move that into its own function that's called by both `handleSubmit` and `handleNotifyAllConfirmation`? Something like
```javascript
handleNotifyAllConfirmation() {
    hideModal();
    doSubmit();
}

handleSubmit() {
    if (message.contains('@all')) {
        showModal();
        return;
    }
    doSubmit();
}

doSubmit() {
    const post = {...}; // see next comment
    sendMessage(post);
}
```
That way, we won't have to keep the mobile keyboard focus logic in sync in multiple places.